.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,3 +30,6 @@ Any special considerations for deployment? Consider both:
 
 - [ ] Testinfra tests pass on the development VM
```

Comment:
Discussed this in person with @redshiftzero and @conorsch. We agreed that this recommendation has the potential to be misleading/confusing, because a significant portion of the system configuration is *not* applied in the development VM, and in many cases you would need to ensure the testinfra tests on passing on one of the staging machines.

For example, in testing #1777, the testinfra tests passed on the development VM because #1777 primarily modified the Apache configuration, which is not applied in the development VM (but it applied in the app-staging VM).

We agreed that the most parsimonious way to resolve this potential confusion in the context of this PR is to change the wording of this checklist item to something like the following:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass